src/mediapipe_calculators/modelapiovmsadapter.cc

```diff
@@ -77,7 +76,7 @@ OVMSInferenceAdapter::OVMSInferenceAdapter(const std::string& servableName, uint
 }
 
 OVMSInferenceAdapter::~OVMSInferenceAdapter() {
-    LOG(ERROR) << "OVMSAdapter destr";
+    LOG(INFO) << "OVMSAdapter destr";
```

Comment:
I guess we would remove that one eventually?

Reply:
Yes this should be removed before release, but it can be helpful during debug when integrating.